easybuild/easyconfigs/s/SNAPE-pooled/SNAPE-pooled-r32_fix-hardcoding-Makefile.patch

```diff
@@ -0,0 +1,12 @@
+fix hardcoded 'gcc' + add optimization flags through $CFLAGS
+author: Kenneth Hoste (HPC-UGent)
+--- snape-pooled/trunk/Makefile.orig	2016-03-10 10:23:16.000000000 +0100
++++ snape-pooled/trunk/Makefile	2016-03-10 10:23:32.000000000 +0100
+@@ -8,6 +8,6 @@
+ 	latex $<
+ 	dvips -o snape-pooled-manual.ps snape-pooled-manual.dvi
+ snape-pooled-fast: snape-pooled.c
+-	gcc -g -o $@ $< -lm
++	${CC} ${CFLAGS} -g -o $@ $< -lm
```

Comment:
I like how they call the binary 'fast', yet they don't use any optimisation flags (and thus they default to `-O0` with GCC)...